### PR TITLE
Refactor data structure for mapobject description #107

### LINKED ISSUE #107: Data structure for Description in MapObject
Description inside the MapObject contains logic that the string cannot describe so easily. It is necessary to create a data structure that will separate meaningful parts from Description.

### PR DESCRIPTION
**Pull request by denisfruza**
Map objects now have Description with its BuildingId, Floor Number and Information instead of all being stored in Description string.
Fix #107

### CODE REVIEW

File: WPFHospitalEditor/AdditionalInformation.xaml.cs [hunk 1]
```diff
@@ -51,11 +50,11 @@ private void Cancel_Click(object sender, RoutedEventArgs e)
 
         private void Done_Click(object sender, RoutedEventArgs e)
         {
-            mapObject.Description = mapObject.Description.Split("&")[0] + "&" + dynamicGridControl.GetAllContent();
+            mapObject.MapObjectDescription.Information = dynamicGridControl.GetAllContent();
             mapObject.Name = this.Title.Text;
             mapObject.nameOnMap.Text = mapObject.Name;
             UpdateAdditionalInformation();
-            mapObject.Description = mapObject.Description.Substring(0, mapObject.Description.Length - 1);
+            //mapObject.MapObjectDescription.Information = mapObject.MapObjectDescription.Information.Substring(0, mapObject.MapObjectDescription.Information.Length - 1);
```
Comment (rammba) [inline WPFHospitalEditor/AdditionalInformation.xaml.cs:57]: Delete comment?

**Comment by rammba** [comment]
[(0) PR Refactor data structure for mapobject description](https://trello.com/c/hqRGLSVf/125-0-pr-refactor-data-structure-for-mapobject-description)

File: WPFHospitalEditor/AdditionalInformation.xaml.cs (hunk 1 above)
Comment (denisfruza) [reply WPFHospitalEditor/AdditionalInformation.xaml.cs:57]: Fixed

File: WPFHospitalEditor/Building.xaml.cs [hunk 2]
```diff
@@ -16,8 +16,8 @@ namespace WPFHospitalEditor
     /// </summary>
     public partial class Building : Window
     {
-        private Dictionary<int, Floor> buildingFloors = new Dictionary<int, Floor>();
-        public List<MapObject> floorBuildingObjects;
+        public Dictionary<int, Floor> buildingFloors = new Dictionary<int, Floor>();
+        //public List<MapObject> floorBuildingObjects;
```
Comment (rammba) [inline WPFHospitalEditor/Building.xaml.cs:20]: You left comments again, here and on 33rd line.

Comment (denisfruza) [reply WPFHospitalEditor/Building.xaml.cs:20]: Will be fixed after Boža's review

**Review by rammba — APPROVED** [review]

**Review by BoKna031 — APPROVED** [review]

**Pull request merged**